packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart

```diff
@@ -21,7 +24,10 @@ typedef EditArgumentFunction =
     });
 
 class PropertyEditorController extends DisposableController
-    with AutoDisposeControllerMixin {
+    with
+        AutoDisposeControllerMixin,
+        SearchControllerMixin<EditableProperty>,
```

Comment:
we should no longer need the search controller mixin here if we are just supporting filter?

Reply:
I'm currently using it to support:

* [`search` and `searchNotifier`](https://github.com/flutter/devtools/blob/6700aa300d54fd17653cd508d365006ffd9b4364/packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart#L57)
* [`currentDataToSearchThrough`](https://github.com/flutter/devtools/blob/6700aa300d54fd17653cd508d365006ffd9b4364/packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart#L98)

I was modeling this off of the `logging_controller` which I think is doing the same thing? (e.g.: https://github.com/flutter/devtools/blob/87f8016e2610c98c3e2eae8b1c823de068701dfd/packages/devtools_app/lib/src/screens/logging/logging_controller.dart#L732)

Reply:
logging controller actually does support search and filter, which is why it has both mixins. This is just filter so you just need the filter mixin. The filter text field is controlled by the filter controller's activeFilter: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/ui/filter.dart/#L654.

When the text field is updated, the activeFilter value is set with the current query:
https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/ui/filter.dart/#L737

Reply:
Ah got it, thank you! This is much neater.